modules/ROOT/pages/partner-manager-create-message-type.adoc

```diff
@@ -60,37 +59,38 @@ Only segment level customization is supported.
 [[target-at-host]]
 == Target at Host
 
-These are application message structure (JSON or XML) to which inbound EDI transactions are translated into and sent to your back-end applications.
+The application message structure (JSON or XML) to which inbound EDI transactions are translated and sent to your back-end applications.
```

Comment:
Reword to be a complete sentence and to introduce the table.

Reply:
Done